include/aspect/utilities.h

```diff
@@ -551,10 +551,35 @@ namespace aspect
     /**
      * Checks whether a file named @p filename exists and is readable.
      *
+     * Warning: This function performs file access on all MPI ranks that
```

Comment:
We are not sure if this is indeed an issue, right? Maybe this warning is a bit too strong?